### PR TITLE
Feat: add mysql operator for airflow scheduler

### PR DESCRIPTION
**Pull request by kaijie-qin**

### CODE REVIEW

File: sqlmesh/schedulers/airflow/operators/mysql.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import typing as t
+
+from airflow.models import BaseOperator
+from airflow.providers.mysql.hooks.mysql import MySqlHook
+from airflow.utils.context import Context
+
+from sqlmesh.schedulers.airflow.operators.targets import BaseTarget
+
+
+class SQLMeshMysqlOperator(BaseOperator):
```
Comment (georgesittas) [inline sqlmesh/schedulers/airflow/operators/mysql.py:12]: Nit - let's match existing naming convention for MySQL:
```suggestion
class SQLMeshMySQLOperator(BaseOperator):
```

**Review by georgesittas — COMMENTED** [review]
Hey @kaijie-qin, thanks for the PR! A couple of things I found to be missing:

- Update the `discover_engine_operator` utility [function](https://github.com/TobikoData/sqlmesh/blob/main/sqlmesh/schedulers/airflow/util.py#L121) by adding case for MySQL
- Update MySQL's [documentation](https://sqlmesh.readthedocs.io/en/stable/integrations/engines/mysql/) to include the Airflow scheduler integration details

It may be helpful to use another PR as a reference for this, e.g. take a look at https://github.com/TobikoData/sqlmesh/pull/2188. Have you tested to make sure this works?

cc @izeigerman can you please take a look as well?

File: sqlmesh/schedulers/airflow/operators/mysql.py (hunk 1 above)
Comment (kaijie-qin) [reply sqlmesh/schedulers/airflow/operators/mysql.py]: just updated the code, I want to know how could I test this operator with airflow in local ?

**Review by georgesittas — APPROVED** [review]
LGTM, @kaijie-qin should be good to go once the [CLA](https://github.com/TobikoData/sqlmesh/pull/2843#issuecomment-2198969658) is signed.

**Pull request merged**